$OpenBSD: patch-meson_build,v 1.2 2017/06/22 20:09:44 jasper Exp $

- don't enter tests/ for it breaks the build atm:
  ../gnome-recipes-1.4.4/tests/../src/gr-ingredient.c:36:30: fatal error: no-ingredients.inc: No such file or directory

Index: meson.build
--- meson.build.orig
+++ meson.build
@@ -117,7 +117,6 @@ subdir('tools')
 subdir('data')
 subdir('po')
 subdir('po-data')
-subdir('tests')
 subdir('help')
 
 meson.add_install_script('meson_post_install.py')
